kinds/structured/preferences/preferences_kind.cue

```diff
@@ -0,0 +1,37 @@
+package kind
+
+name: "Preferences"
+maturity: "merged"
+
+lineage: seqs: [
+	{
+		schemas: [
+			{//0.0
+				// UID for the home dashboard
+				homeDashboardUID?: string
+
+				// The timezone selection
+				// Would be nice it this used:
+				// import { TimeZone } from '@grafana/data';
+				timezone?: string
```

Comment:
@sdboyer -- is it possible to get this to be TimeZone in typescript?   The raw type definition is pretty simple, but good to have it identified as `TimeZone` when using it:

https://github.com/grafana/grafana/blob/main/packages/grafana-data/src/types/time.ts#L35

Reply:
we haven't yet broken the seal on allowing imports of the mudball from core types, but i see no reason why we couldn't. it would mean moving `TimeZone` into `packages/grafana-schema/src/schema/mudball.cue`, though, and importing it from there

At that point you can either directly import and reference it, or use a TS veneer and supersede the type there